bigbluebutton-html5/imports/ui/components/chat/notification/component.jsx

```diff
@@ -113,13 +113,16 @@ class ChatNotification extends Component {
       disableNotify,
       openChats,
       intl,
+      chatIsOpen,
+      currentChatID,
     } = this.props;
 
     const chatsNotify = openChats
       .filter(({ id, unreadCounter }) =>
```

Comment:
Could you split this filter in multiple filter or at least make it more readable by moving the conditions to variables?

Reply:
Ok